metadata-ingestion/src/datahub/ingestion/run/pipeline.py

```diff
@@ -465,10 +465,10 @@ def log_ingestion_stats(self) -> None:
             self.ctx.graph,
         )
 
-    def _count_all_vals(self, d: Dict[str, List]) -> int:
-        result = 0
-        for val in d.values():
-            result += len(val)
+    def _approx_all_vals(self, d: LossyDict[str, LossyList]) -> int:
+        result = d.overflow
+        for k in d:
+            result += len(d[k])
```

Comment:
this is tricky, because the keys of d are subject to replacement, and we don't have a way to hook into the eviction logic to keep track of all the elements in all the LossyLists that are past and current members of this dictionary.

Reply:
`sum(d[k].total_elements) for k in d` will just give us the total elements for the lists that have survived.